docs/install/osx_setup.md

```diff
@@ -65,14 +65,19 @@ Install the dependencies, required for MXNet, with the following commands:
 	brew install openblas
 	brew tap homebrew/core
 	brew install opencv
-	# For getting pip
+	# Install Python (if you already have python installed, use "brew reinstall python" instead)
 	brew install python
+	# Get pip
+	easy_install pip
 	# For visualization of network graphs
 	pip install graphviz
 	# Jupyter notebook
 	pip install jupyter
 ```
 
+#### Note
+If you already have Python installed on your Mac, you may face the issue while installing jupyter caused by restricted access in System(see [here](https://stackoverflow.com/questions/33004708/osx-el-capitan-sudo-pip-install-oserror-errno-1-operation-not-permitted)). In that case, use brew resintall python to install another version of Python under homebrew. Use pip2 instead of pip to install the following packages. It also requires you to run a python program using python2 (e.g $ python2 example.py) instead of python.
```

Comment:
Typo in: use "brew **resintall** python" to install another version of Python under homebrew.

Reply:
Done! Make another commit to fix that issue.